pyacq/core/stream/ringbuffer.py

```diff
@@ -215,8 +215,15 @@ def get_data(self, start, stop, copy=False, join=True):
         copied = False
         
         if self.double:
-            start_ind = start % bsize
-            stop_ind = start_ind + (stop - start)
+            # This do not work when get_data(-10, 50) meaning stop=50 length=60 (start=stop-length)
```

Comment:
It seems like these only give different results if you request a data chunk that is larger than the buffer. In that case, we should probably just raise an exception anyway, right?